development/tools/cmd/gcscleaner/main_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func cleanFlags() {
+	argProjectName = ""
+	argBucketLifespanDuration = bucketLifespanDurationDefault
+	argExcludedBucketNames = ""
+}
+
+func TestConfigRead(t *testing.T) {
+	tests := []struct {
```

Comment:
Why exactly we should do that?

Reply:
To group success and error cases, without need to add sufix like `error` and `pass`, which doesn't look nice.
It would look like this:
```
func Test...(...) {
 t.Run(...) {
   t.Run(...) {
   }
}
}
```

Reply:
I understand the idea. My question was rather why should we have an explicit division between `Success` and `Failure` and how this division would improve current implementation?

Reply:
It's just for better readability, when executing tests. I see that you even went one step further and removed all names from test cases. I can't agree with this approach as it won't tell much on failed test.

The test case name is to describe it.

Reply:
Also we discussed f2f that you would like BDD approach and you will use GoConvey for that (existing dependency). I don't see the changes.

Reply:
The tests are checking how the read function handles conversion between passed arguments and cleaner configuration. The most important thing in my opinion is if they are covering possible cases rather than if we use standard library, package A or package B.
The other (also important) thing is that the tests should be readable enough, and again in my opinion it is OK.